plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouterArgs.sol

```diff
@@ -16,17 +16,17 @@ library PaymentStandardExitRouterArgs {
     }
 
     /**
-     * @notice Input args data for challengeStandardExit.
-     * @param exitId Identifier of the standard exit to challenge.
-     * @param exitingTx The transaction that is exiting.
-     * @param challengeTx RLP encoded transaction that spends the exiting output.
-     * @param inputIndex Which input of the challenging tx corresponds to the exiting output.
-     * @param witness Witness data that can prove the exiting output is spent.
-     * @param spendingConditionOptionalArgs optional extra data for the spending condition.
-     * @param outputGuardPreimage (Optional) output guard preimage for the challenge tx to use the output
-     * @param challengeTxPos (Optional) tx position of the challenge tx if it is of MVP protocol.
-     * @param challengeTxInclusionProof (Optional) if the challenge tx is of MVP protocol provide the inclusion proof of it
-     * @param challengeTxConfirmSig (Optional) if the challenge tx is of MVP protocol provide the confirm sig of it
+     * @notice Input args data for challengeStandardExit
+     * @param exitId Identifier of the standard exit to challenge
+     * @param exitingTx The exiting transaction
```

Comment:
let's be consistent with the next line:

`RLP-encoded transaction that creates the exiting output`